index.bs

```diff
@@ -338,7 +338,7 @@ brand of the browser to the user. These messages often also include the platform
 order to make sure the user knows which device is in question.
 
 Since such messaging doesn't require any server-side adaptation, it's better for this case to use
-the `userAgentData.getHighEntropyData()` method in order to retrieve the required information.
+the `userAgentData.getHighEntropyValues()` method in order to retrieve the required information.
```

Comment:
Is this rename web compatible?

Reply:
Good question. I was also a bit confused when I saw that :)

Turns out we just had 2 instances of _the wrong name_ in the use-cases section (🙈). Not sure when that crept it, or if getHighEntropyData was the original name way back when. But we're not actually changing any IDL - `getHighEntropyValues` is the thing we shipped (and continue to ship).